model-optimizer/extensions/front/MatMul_normalizer.py

```diff
@@ -57,7 +57,7 @@ def replace_sub_graph(self, graph: Graph, match: [dict, SubgraphMatch]):
                                          new_op_attrs={'name': name + '/weights_transpose'}, value=int64_array([1, 0]))
 
         # input normalization for 4D Caffe and MxNet FullyConnected
-        if graph.graph['fw'] in ['caffe', 'mxnet']:
+        if graph.graph['fw'] in ['caffe']:
```

Comment:
Why MxNet don't need input normalization now? If it is correct, please update comment in previous line